nitf/nitf_lib.ddl

```diff
@@ -141,126 +155,126 @@ def Leq x y = (Eq x y) | (Lt x y)   -- XXX: Why not use <= ?
 
 def PartialEq (x : OrBytes) (y : OrBytes) =
```

Comment:
In principle, it seems better to make this a pure computation, but it's not essetial.